salt/output/json_out.py

```diff
@@ -54,7 +54,7 @@ def output(data):
     '''
     try:
         if 'output_indent' not in __opts__:
-            return json.dumps(data, default=repr, indent=4)
+            return json.dumps(data, default=repr, indent=4, encoding='latin1')
```

Comment:
It is only for pkg.list_pkgs. I can try to get the values which cause the issue. 
As in the commit description the issue is on 0x82 which is a control value in utf8.